flixel/ui/FlxAnalog.hx

```diff
@@ -90,25 +91,26 @@ class FlxAnalog extends FlxSpriteGroup
 	/**
 	 * Create a virtual thumbstick - useful for input on mobile devices.
 	 *
-	 * @param   X        The X-coordinate of the point in space.
-	 * @param   Y        The Y-coordinate of the point in space.
-	 * @param   radius   The radius where the thumb can move.
-	 *                   If `0`, half the background's width will be used as radius.
-	 * @param   ease     The duration of the easing. The value must be between `0` and `1`.
+	 * @param   X            The X-coordinate of the point in space.
+	 * @param   Y            The Y-coordinate of the point in space.
+	 * @param   Radius       The radius where the thumb can move. If 0, half the base's width will be used.
+	 * @param   Ease         Used to smoothly back thumb to center. Must be between 0 and (FlxG.updateFrameRate / 60).
+	 * @param   BaseGraphic  The graphic you want to display as base of the joystick.
+	 * @param   ThumbGraphic The graphic you want to display as thumb of the joystick.
 	 */
-	public function new(X:Float = 0, Y:Float = 0, Radius:Float = 0, Ease:Float = 0.25)
+	public function new(X:Float = 0, Y:Float = 0, Radius:Float = 0, ?Ease:Float = 0.25, ?BaseGraphic:FlxGraphicAsset, ?ThumbGraphic:FlxGraphicAsset)
```

Comment:
I don't think `Ease` shouold have a `?`, that unnecessarily turns it into a `Null<Float>`.